org.kde.kmahjongg.json

```
@@ -0,0 +1,69 @@
+{
+    "id": "org.kde.kmahjongg",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kmahjongg",
+    "rename-icon": "kmahjongg",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=fallback-x11",
```

```suggestion
        "--socket=wayland",
        "--socket=fallback-x11",
```

`--socket=fallback-x11` is enough